### ISSUE TITLE
Support NullBooleanField

### ISSUE THREAD

**Issue by berinhard** [vandersonmota/model_mommy#305]
I received this exception:

```
E           TypeError: <class 'django.db.models.fields.NullBooleanField'> is not supported by mommy.
```

**Comment by robertfw** [comment]
Should be an easy fix in generators.py & random_gen.py

Not setup atm to pull down and put a pull request in but the following does the trick:

top of generators.py - add NullBooleanField to imports
```python
from django.db.models import (
    CharField, EmailField, SlugField, TextField, URLField,
    DateField, DateTimeField, TimeField,
    IntegerField, SmallIntegerField, PositiveIntegerField,
    PositiveSmallIntegerField, BooleanField, DecimalField,
    FloatField, FileField, ImageField, IPAddressField,
    ForeignKey, ManyToManyField, OneToOneField, NullBooleanField)
```
a little further down, add entry to default_mapping
```python
default_mapping = {
    ForeignKey: random_gen.gen_related,
    OneToOneField: random_gen.gen_related,
    ManyToManyField: random_gen.gen_m2m,

    BooleanField: random_gen.gen_boolean,
    NullBooleanField: random_gen.gen_nullboolean,
    #...
}
```

then in random_gen.py
```
def gen_nullboolean():
    return choice((True, False, None))
```

**Comment by robertfw** [comment]
As a workaround without modifying the source, I'm using the following. I already had a wrapper around mommy.make to let me set some usual defaults and isolate the 3rd party code, so I added a quick fix there

```python
def make(obj, **overrides):
    defaults = {
        'make_m2m': True,
        '_fill_optional': True
    }

    defaults.update(overrides)

    # NullBooleanFields are not supported so we need to handle those ourself
    for field in obj._meta.fields:
        if isinstance(field, fields.NullBooleanField):
            defaults[field.name] = random.choice([None, True, False])

    return mommy.make(obj, **defaults)
```

**Comment by vandersonmota** [comment]
fixed: 25b0f2d
